tests/unit/threads/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ set(tests
     thread_launching
     thread_mf
     thread_stacksize
+    thread_stacksize_overflow
     thread_suspension_executor
     thread_yield
    )
```

Comment:
@sithhell thank you for the suggestion. @hkaiser will be sure to provide a platform check.